src/main/scala/abalone/format/FEN.scala

```diff
@@ -30,32 +30,36 @@ final case class FEN(value: String) extends AnyVal {
 
   def player2Score: Int = intFromFen(2).getOrElse(0)
 
-  def prevPlayer(variant: Variant): Option[Player] = {
-    if (!variant.hasPrevPlayer) return None
-
-    value
-      .split(' ')
-      .lift(4)
-      .flatMap(_.headOption)
-      .flatMap(Player.apply)
-      .map(!_)
-  }
-
   def player: Option[Player] = value
     .split(' ')
     .lift(3)
     .flatMap(_.headOption)
     .flatMap(Player.apply)
     .map(!_)
 
-  def halfMovesSinceLastCapture(variant: Variant): Option[Int] = intFromFen(
-    if (variant.hasPrevPlayer) 5 else 4
-  )
+  def halfMovesSinceLastCapture: Option[Int] = intFromFen(4)
 
-  def fullMove(variant: Variant): Option[Int] = intFromFen(if (variant.hasPrevPlayer) 6 else 5)
+  def fullMove: Option[Int] = intFromFen(5)
+
```

Comment:
I dont think we need to put lastTurn into the FEN?

To keep consistency, we just want currentTurn to be put in after the ½ character.

This way we don't even need to pass through the variant we can just see is there is a ½ in the fen and build up the currentTurn from that. I prefer not needing to pass the variant into functions in the FEN class.

Amazons does something like:
`Uci.Move.apply(variant.gameFamily, fen.value.dropWhile('½' !=).drop(1))` which might also work here.